airflow/ui/src/components/ParseDag.tsx

```diff
@@ -30,10 +31,19 @@ const ParseDag = ({ dagId, fileToken }: Props) => {
   const { isPending, mutate } = useDagParsing({ dagId });
 
   return (
-    <Button loading={isPending} onClick={() => mutate({ fileToken })} variant="outline">
-      <AiOutlineFileSync height={5} width={5} />
-      Reparse Dag
-    </Button>
+    <Box>
```

Comment:
Nit, do we need that extra `Box` component ?

Reply:
I attempted to remove the box but then... the buttons were presented horizontal:
![image](https://github.com/user-attachments/assets/5466d061-2245-4a58-ad0f-2adbaeaaea84)

I looked at the docs in https://www.chakra-ui.com/docs/components/menu but somehow our modelling of menu is different than the standard - don't know if this is a "feature" (=by intend) or a "bug".

Reply:
Weird, is that from the latest branch or a specific browse ? If I try on my end I don't see the issue after removing the box of `ParseDag`:
![Screenshot 2025-03-03 at 14 51 09](https://github.com/user-attachments/assets/08975018-2625-4ae7-b4d6-aa20946ca63a)


Anyway that might come from the `display` prop of the button. If you play with the `display: flex` that should be enough for this to work. (You might end-up with `flex-inline` or something).

Non blocking.

Reply:
Looking good on my end. Is it still horizontal for you, Jens?

Reply:
It is getting horizontal if I remove the `Box` from the Backfill :-D